x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/gap_auto_fill_schedule/create/create_auto_fill_scheduler_route.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { IRouter } from '@kbn/core/server';
+import type { GapAutoFillSchedulerResponseV1 } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import { gapAutoFillSchedulerBodySchemaV1 } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import type { ILicenseState } from '../../../../../lib';
+import { verifyAccessAndContext } from '../../../../lib';
+import type { AlertingRequestHandlerContext } from '../../../../../types';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../../../../../types';
+import { transformRequestV1, transformResponseV1 } from './transforms';
+import { DEFAULT_ALERTING_ROUTE_SECURITY } from '../../../../constants';
+
+export const createAutoFillSchedulerRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.post(
+    {
+      path: `${INTERNAL_BASE_ALERTING_API_PATH}/rules/gaps/gap_auto_fill_scheduler`,
```

Comment:
nit: should this use the `INTERNAL_ALERTING_GAPS_API_PATH` defined in `x-pack/platform/plugins/shared/alerting/common/index.ts`?

Since the route already has `gaps` in it, what do you think about calling it `_schedule_autofill` like `internal/alerting/rules/gaps/_schedule_autofill`?

Reply:
Updated path. 

About rename. In the future we want to support update/delete methods, so what about

`gaps/auto_fill_scheduler`

POST - create
PUT - update
DELETE -delete